### PR TITLE
add beaker bindings to terarium notebooks

### PR DESCRIPTION
**Pull request by blanchco**
# Description

* Adding beaker keybindings to terarium notebook
`j`, `ArrowUp` -> select previous cell
`k`, `ArrowDown` -> select next cell
`a` -> add cell above
`b` -> add cell below
`enter` -> enter cell
`esc` -> exit cell

* fix: default dataset cell when opening notebook

### CODE REVIEW

**Review by shawnyama — COMMENTED** [review]
getting some overflow issue
image

File: packages/client/hmi-client/src/components/llm/tera-jupyter-chat.vue [hunk 1]
```diff
@@ -374,8 +411,12 @@ watch(
 	() => {
 		if (props.notebookSession) {
 			notebookItems.value = props.notebookSession.data.history;
+			if (!notebookItems.value.length) {
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/llm/tera-jupyter-chat.vue:414]: use `isEmpty`

**Review by shawnyama — APPROVED** [review]

**Comment by blanchco** [comment]
> getting some overflow issue image

Hmmm, not really getting anything different on my side, I didnt really touch the css or templating.  Let me see if I can create something similar

**Comment by blanchco** [comment]
The overflow issue was present before these changes, will fix this up when Im working on the cells

**Review by asylves1 — APPROVED** [review]

**Pull request merged**